spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/converters/DataConverter.scala

```diff
@@ -83,7 +83,7 @@ object DataConverter extends DataConverter{
     : universe.Type = {
     if (value.isInstanceOf[Tensor[_]]) {
       ModuleSerializer.tensorType
-    } else if (value.isInstanceOf[AbstractModule[_, _, T]]) {
+    } else if (value.isInstanceOf[AbstractModule[_, _, _]]) {
```

Comment:
Please take a look at the change in the file @wzhongyuan

Reply:
looks good